bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/welcome/NAWelcomeHomeHandler.java

```diff
@@ -145,7 +146,7 @@ protected State getNAThingProperty(String channelId) {
                     String cameraId = lastEvent.get().getCameraId();
                     Optional<AbstractNetatmoThingHandler> thing = getBridgeHandler().findNAThing(cameraId);
                     if (thing.isPresent()) {
-                        NAWelcomeCameraHandler eventCamera = (NAWelcomeCameraHandler) thing.get();
+                        CameraHandler eventCamera = (CameraHandler) thing.get();
```

Comment:
I might have merged too quickly, but just to double check, I don't see you changing NAWelcomeCameraHandler to extend CameraHandler. Was that supposed to be done?

Reply:
Yes, that is fine. I introduced CameraHandler with a previous pull-request and NAWelcomeCameraHandler and NAPresenceCameraHandler are extending for CameraHandler. It was rather a bug that there was still casted to NAWelcomeCameraHandler (but it was no problem so far, because the Presence was also represented as a NAWelcomeCamera still this new pull-request).